docs/fast_data/what_is_fast_data.md

```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
+
+* Monitor the status of various runtime pipelines.
+* Using a dedicated frontend, users can resume or stop pipeline components by sending commands to the underlying microservices.
+* Collect feedback from workloads to verify their operational status within the associated pipelines.
+
+For more information about Control Plane configuration, visit **[this page](/fast_data/runtime_management/control_plane.mdx)**. 
+
+## Fast Data to accomplish your Data Integration Hub
+
+Mia-Platform allows you to set up and deploy your end-to-end **Digital Integration Hub-enabled API platform**, as mentioned in the **[Gartner® Hype Cycle™ for Application Architecture and Integration, 2022](https://mia-platform.eu/news-en/gartner-application-architecture-and-integration/)**.
+
+![Mia-Platform Fast Data DIH architecture](img/fastdata-DIH-architecture.png)
+
+Digital Integration Hub is an important architectural component offering low-latency, high-throughput data access via APIs and event-based mechanisms. It separates SoRs sources from external channels through a data management layer that ensures data synchronization using event-driven patterns.
+Key benefits are:
+* Access aggregated real-time data scattered across SoRs.
+* Upgrade legacy systems with minimal disruption to the API layer.
+* Ensure continuous data provisioning 24/7, even when some APIs are temporarily unavailable.
+* Maintain data availability 24/7, even when systems are offline for scheduled maintenance.
+* Serve channels with a persistent data layer, ensuring access even during system downtimes.
+* Reduce the load on costly SoRs by offloading tasks to API services.
+* Facilitate data integration by forwarding DIH data to other endpoints.
+* Offer real-time connectors for data analytics and business intelligence (BI) applications.
+
 ## How is Fast Data different from similar products?
 
-"ETL" systems (Extract, Transform, Load) are not big news, they have been around for decades and their principles are now a staple in the industry. So what makes Fast Data stand out from the crowd? It is easy to set up thanks to its **low code/ no code** functionality, fully customizable injecting actual javascript code as configuration, and part of an **ecosystem** that will grant you all the tools you need to make the most out of your standardized and aggregated data.
+"ETL" systems (Extract, Transform, Load) have been industry standards for decades, forming a core component of data management.
+What differentiates Mia-Platform Fast Data is its ease of setup through **No Code / Low Code** functionality, full customizability via JavaScript code as configured, and its integration within an **ecosystem** that provides all necessary tools for maximizing standardized and aggregated data.
+
+Moreover, you can build your Data Product utilizing a suite of tools specifically engineered to optimize Data acquisition and pipeline efficiency, for example: 
```

Comment:
On the following bullet points, it could be useful to provide a link to each tool/feature listed below

Reply:
Hi Alberto, ok i provide a link to each tool/feature. Thanks!